flux_led/__main__.py

```diff
@@ -676,28 +725,37 @@ def __str__(self):
           mode_str += str(_r) + ","
         return "{} [{}]".format(power_str, mode_str)
 
-    def turnOn(self, retry=2):
-        self._is_on = True
-        msg = bytearray([0x71, 0x23, 0x0f])
+
+    def _change_state(self, retry, turn_on = True):
+        msg_on = bytearray([0x71, 0x23, 0x0f])
+        msg_off = bytearray([0x71, 0x24, 0x0f])
+        if self.protocol == 'LEDnet_wifi370':
+            msg_on =  bytearray([0xcc, 0x23, 0x33])
+            msg_off =  bytearray([0xcc, 0x24, 0x33])
+
```

Comment:
I thought it would be cleaner to define the default cases first and then define the special cases (such as where we use the original protocol) after that. I changed the order.